app/home/controllers.js

```diff
@@ -34,8 +36,14 @@ function dashboard(req, res) {
     },
   }
 
+  const [fromDate, toDate] = dateHelpers.getCurrentWeekAsRange()
+
   res.render('home/dashboard', {
     pageTitle: 'dashboard::page_title',
+    week: `(${formatters.date(fromDate, 'd MMM')} - ${formatters.date(
```

Comment:
It's best to use `to` instead of dashes in dates as per the [GOV.UK styleguide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates).

Reply:
You can also just use [this nunjucks filters](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/ba14c1c25c69ba56750fd2b18ead7c3d4df82345/config/nunjucks/filters.js#L68) which already supports the use of `to` and will shorten the dates if they contain the same month or year.